auth-svc/prisma/schema.prisma

```diff
@@ -15,13 +15,13 @@ model tokens {
 }
 
 model users {
-  user_id        Int      @id @default(autoincrement())
+  user_id        Int      @id @unique @default(autoincrement())
```

Comment:
we can use uuid() to create a random id instead of auto incrementing . 

also can you explain the activationLink? i am assuming thats how the user would activate their account but what is the purpose of activating an account vs simply creating one? 

everything else looks good to me!

Reply:
Yeah, uuid is a great idea. This model is just a prototype. We will think it through when we talk about DB models.
As well as the activation link, right now it's kind of a useless feature, but in the future, we can add the feature to verify users by email and make it required during sign-up process. So we will also rebuild this functionality.